.github/workflows/custom-test.yml

```diff
@@ -211,3 +211,37 @@ jobs:
               --certificate-identity $IDENTITY \
               --certificate-oidc-issuer https://token.actions.githubusercontent.com \
               artifact
+
+  sigstore-rs:
+    runs-on: ubuntu-latest
+    needs: [sigstore-python]
+    steps:
+      # checkout branch of sigstore-rs with root v12 embedded
+      # TODO: Just use a released sigstore-rs when possible
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          repository: "jku/sigstore-rs"
```

Comment:
Would your change need to be merged into upstream sigstore-rs after this so we can depend on that instead?

Reply:
Yes. I would expect 
* root v12 change gets merged in sigstore-rs after we publish
* sigstore-rs makes a release
* we can start using the released sigstore-rs (maybe even via cargo and not actions checkout)